corehq/apps/aggregate_ucrs/models.py

```diff
@@ -153,5 +153,5 @@ class SecondaryColumn(models.Model):
     table_definition = models.ForeignKey(SecondaryTableDefinition, on_delete=models.CASCADE,
                                          related_name='columns')
     column_id = models.CharField(max_length=MAX_COLUMN_NAME_LENGTH)
-    aggregation_type = models.CharField(max_length=10, choices=SECONDARY_COLUMN_TYPE_CHOICES)
+    aggregation_type = models.CharField(max_length=12, choices=SECONDARY_COLUMN_TYPE_CHOICES)
```

Comment:
Any reason to not bump this up higher? Keeping it artificially low might encourage some funky naming later or some migration noise later.

Alternatively you could use an integer column and keep this more like an enumeration instead of text (may make yml -> object translation messier though).

Reply:
makes sense. i have a bit of an aversion to numeric codes in this day and age where clarity >> disk space, so bumped to 20.